### DISCUSSION TITLE
Can i use this theme with it?

### DISCUSSION THREAD

**Discussion by pixelsurium** [vladmandic/sdnext#75] [Q&A]
https://github.com/canisminor1990/sd-web-ui-kitchen-theme.git

**Comment by vladmandic** [comment]
since that theme is written to overwrite the `style.css`, you'd need to remove `user.css` which ships as part of this repo.
then it should work.

**Reply by RaXorX** [reply]
hi, can't really seem to find user.css anywhere. does this theme still work with this repo?

**Reply by Tillerz** [reply]
user.css should be in automatic/. If it's not there, just try the theme now. :)

**Reply by RaXorX** [reply]
@Tillerz @vladmandic user.css is nowhere to be found. In javascript, there's styles.css, but that seems crucial to the UI itself. I think that's what's making it a bit broken. No, I've already tried the theme but it's a bit weird in this repo. Plus the Additional Network pane, it doesn't align the elements inside it properly. 

Everything actually works except the additional networks pane. 

![image](https://github.com/vladmandic/automatic/assets/44987810/66e87135-da92-4bae-b90e-2d0b53c3971d)

It'd be cool if the fork repo had something that made it back to SD's UI, maybe that might work properly. This happens with Kitchen as well unlike the user below. 

EDIT: Yep, emptied the styles.css entirely and that worked. :) IDK why it's apparent for the UI to launch though, I can't delete it but emptying it worked fine. 

![image](https://github.com/vladmandic/automatic/assets/44987810/4d91d629-f4c9-4faa-a31d-1ba3e112b41f)

It's better now. Only if I could just fix the Extra Network pane, the prompts text area and the image result window at the bottom right.

**Comment by Dragoy** [comment]
Perhaps one of the most beautiful and convenient themes for automatic, I would really like to have native support for this theme in your fork. 

I installed this theme, removed style.css from the main automation folder, switched the gardio theme to default and have the following picture, what else can I do to make it display correctly?
<img width="1920" alt="image" src="https://user-images.githubusercontent.com/4968313/235358468-d0c6c066-7df9-467b-bc30-9e47a3a6692a.png">

**Comment by vladmandic** [comment]
a) do not delete or modify style.css. that is a system file and will be overwritten during upgrade
b) any modifications you want to make to style.css, make in user.css. yes, that file does not exist by default as its just empty. you create it in root folder and put in it anything you want. it serves as override.
c) i have not looked at kitchen theme in a long time, just don't have the time to follow everything

**Reply by Dragoy** [reply]
I think you should stick to the standard design that is used in the main repository automatic in this case the user will be a little more flexible in the choice of design and personalization will not conflict with your version

**Reply by vladmandic** [reply]
I disagree.

**Reply by Dragoy** [reply]
I tested your hack with adding an empty user.css, it still doesn't overwrite the original style.css. Clearing style.css doesn't produce any results either
<img width="960" alt="image" src="https://github.com/vladmandic/automatic/assets/4968313/9ded4472-fc84-4597-8eac-77c7f33bc41d">

**Reply by vladmandic** [reply]
empty user.css doesn't do anything. user.css doesn't REPLACE styles.css, its applied ON TOP OF IT. 
so you can override every value you want individually - do that instead of editing styles.css.
